scrapy/linkextractors/lxmlhtml.py

```diff
@@ -52,6 +52,12 @@ def _iter_links(self, document):
                     continue
                 yield (el, attrib, attribs[attrib])
 
+    def check_for_invalid_sublink (self, url):
+            invalid_url_substring = "../"
+            if invalid_url_substring in url:
```

Comment:
creating this `invalid_url_substring` constant doesn't make sense to me, it is not used consistently (e.g. in replace), not named as a constant (upper-case letters), and doesn't make code more readable. I think it is better to just use '../'.